packages/emails/templates/attendee-was-requested-to-reschedule-email.ts

```diff
@@ -91,7 +91,7 @@ ${this.t("request_reschedule_subtitle", {
 })},
 ${this.getWhen()}
 ${this.t("need_to_reschedule_or_cancel")}
-${getManageLink(this.calEvent)}
+${getManageLink(this.calEvent, this.calEvent.organizer.language.translate)}
```

Comment:
Shouldn't this be the language of the attendee here? 

Also, I tried to test it, and the whole request to reschedule email for the attendee was in the organizer's language. But this doesn't need to be fixed in this PR

Reply:
Let's use `this.t` as in the lines above. And then we still need a follow-up as this email is sent to the attendee but I think the organizer language is used